linetools/isgm/utils.py

```diff
@@ -889,9 +889,10 @@ def joebvp_from_components(comp_list, specfile, outfile):
 
 
 def unique_components(comps1, comps2, tol=5*u.arcsec):
-    """ Identify which members of comps1 are *not* within
-    comps2, to given tolerances.  Note, that repeats in
-    comps1 are not looked for.
+    """ Identify which AbsComponent members of the comps1 list
+    are *not* within the comps2 list, to given tolerances.
+    Note, AbsComponent objects in the comps1 list are examined
```

Comment:
AbsComponent objects in the comps1 list are *not* examined against each other right?  I though that's what the previous docstring was saying.

Reply:
whoops..  edited